IOException when parsing command ALTER DEFAULT PRIVILEGES...

Caused by: java.sql.SQLException: Error on command ' ALTER DEFAULT PRIVILEGES IN SCHEMA xy GRANT SELECT ON TABLES TO username '
    ...
Caused by: java.sql.SQLException: java.io.IOException: error parsing command tag
    at com.impossibl.postgres.jdbc.PGConnectionImpl.execute(PGConnectionImpl.java:452)
    at com.impossibl.postgres.jdbc.PGStatement.executeStatement(PGStatement.java:372)
    at com.impossibl.postgres.jdbc.PGSimpleStatement.execute(PGSimpleStatement.java:105)
    at com.impossibl.postgres.jdbc.PGSimpleStatement.execute(PGSimpleStatement.java:124)
    ... 57 more
Caused by: java.io.IOException: error parsing command tag
    at com.impossibl.postgres.protocol.v30.ProtocolImpl.receiveCommandComplete(ProtocolImpl.java:1118)
    at com.impossibl.postgres.protocol.v30.ProtocolImpl.dispatch(ProtocolImpl.java:833)
    at com.impossibl.postgres.protocol.v30.MessageHandler.channelRead(MessageHandler.java:46)
    at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:332)
    at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:318)
    at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:163)
    at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:332)
    at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:318)
    at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:787)
    at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:125)
    at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:507)
    at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:464)
    at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:378)
    at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:350)
    at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:116)
    ... 1 more

--- a/src/main/java/com/impossibl/postgres/protocol/v30/ProtocolImpl.java
+++ b/src/main/java/com/impossibl/postgres/protocol/v30/ProtocolImpl.java
@@ -1114,6 +1114,10 @@ public class ProtocolImpl implements Protocol {
           command += " " + parts[1];
           rowsAffected = 0L;
         }
+        else if (parts.length == 3) {
+          command += " " + parts[1] + " " + parts[2];
+          rowsAffected = 0L;
+        }
         else {
           throw new IOException("error parsing command tag");
         }

--- a/src/test/java/com/impossibl/postgres/jdbc/StatementTest.java
+++ b/src/test/java/com/impossibl/postgres/jdbc/StatementTest.java
@@ -211,6 +211,12 @@ public class StatementTest {
   }
 
   @Test
+  public void testAlterTable() throws SQLException {
+    Statement stmt = con.createStatement();
+    stmt.execute("ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO pgjdbc");
+  }
+
+  @Test
   public void testNumericFunctions() throws SQLException {
     Statement stmt = con.createStatement();
 

